### PR TITLE
Styling issues of the install options

### PR DESCRIPTION
**Pull request by Jako**
- wrong position of the legend tag
- width: 50% makes no sense

### CODE REVIEW

File: _build/install.options/user.input.php [hunk 1]
```diff
@@ -48,15 +48,13 @@
  */
 
 $templates = $modx->getCollection('modTemplate');
-$output = "<p style=\"width:50%\">The getDynaDescription snippet will allow you to create description Meta tags by placing a tag in your template(s). The snippet can use the value of the dynaDescription Template variable, the description resource field, or the beginning of the resource content for the Meta tag.</p><br />
+$output = "<p>The getDynaDescription snippet will allow you to create description Meta tags by placing a tag in your template(s). The snippet can use the value of the dynaDescription Template variable, the description resource field, or the beginning of the resource content for the Meta tag.</p><br />';
```
Comment (BobRay) [inline _build/install.options/user.input.php]: This line has a double quote at the beginning and a single quote at the end.

File: _build/install.options/user.input.php [hunk 2]
```diff
@@ -48,15 +48,13 @@
  */
 
 $templates = $modx->getCollection('modTemplate');
-$output = "<p style=\"width:50%\">The getDynaDescription snippet will allow you to create description Meta tags by placing a tag in your template(s). The snippet can use the value of the dynaDescription Template variable, the description resource field, or the beginning of the resource content for the Meta tag.</p><br />
+$output = "<p>The getDynaDescription snippet will allow you to create description Meta tags by placing a tag in your template(s). The snippet can use the value of the dynaDescription Template variable, the description resource field, or the beginning of the resource content for the Meta tag.</p><br />';
+$output .= '<p>If you would like to have the dynaDescription template variable created automatically and attached to any of your templates, check the appropriate boxes below. If you plan to use the resource description field or the resource content, you won't need the TV.</p><br />';
```
Comment (BobRay) [inline _build/install.options/user.input.php]: This line contains the word "won't" so it needs double quotes.

**Review by BobRay — COMMENTED** [review]
If you're fixing the quotes, could you change the instances of 0px to just 0?

If you'd rather have me fix everything, that's fine, I just thought that if I fix them, you might face a merge conflict.

**Comment by Jako** [comment]
The only merge conflict would happen on your side. But all 0px changes and the code fixes are done now.

**Comment by Jako** [comment]
This PR seems to be manually merged into your repository.

**Comment by BobRay** [comment]
Yes, it's often easier for me to do it that way.

**Pull request closed without merging**